src/components/ModalViewContainer/index.tsx

```diff
@@ -27,86 +58,40 @@ const useStyles = makeStyles((theme) => ({
     marginLeft: theme.spacing(2),
     marginRight: theme.spacing(2),
     marginTop: theme.spacing(0),
-    marginBottom: theme.spacing(2),
+    marginBottom: theme.spacing(2)
     // paddingLeft: theme.spacing(0),
     // paddingRight: theme.spacing(0),
     // paddingTop: theme.spacing(0),
     // paddingBottom: theme.spacing(0),
   }
-
 }));
 
-
-const ModalViewContainer = (props) => {
+const ModalViewContainer = props => {
   const classes = useStyles();
 
   const routingInfoRef = useRef({});
-  const {
-    getPConnect,
-    routingInfo,
-    loadingInfo
-  } = props;
+  const { getPConnect, routingInfo, loadingInfo } = props;
+  const pConn = getPConnect();
   const {
     CONTAINER_TYPE: { MULTIPLE },
     // eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
```

Comment:
It looks like this eslint directive isn't necessary any more. So it should be removed.